2024-04-02 21:44:33,199 - INFO] DVC training
2024-04-02 21:44:33,199 - INFO] config :
2024-04-02 21:44:33,245 - INFO] {
    "tot_epoch": 10000,
    "tot_step": 2000000,
    "train_lambda": 1024,
    "lr": {
        "base": 0.0001,
        "decay": 0.1,
        "decay_interval": 1800000
    }
}
NUMERO DI RIGHE:  64612
lunghezza train set:  2500
NUMERO DI RIGHE:  7824
lunghezza valid set:  200
NUMERO DI RIGHE:  7824
in test ho esattamente:  54698
lunghezza test set:  50
************************************  0
epoch 0
/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py:3635: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode)
/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py:4004: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  "Default grid_sample and affine_grid behavior has changed "
Train epoch 0: [0/625 (0.0%)]	Loss: 462.235 |	MSE loss: 0.446 |	Bpp loss: 5.30 |	Aux loss: 0.00
epoch 0
epoch 0
Traceback (most recent call last):
  File "main_dvc.py", line 521, in <module>
    _,global_step_test = test(epoch,global_step_test, mode = "test")
  File "main_dvc.py", line 356, in test
    input_image, ref_image = Var(input[0]), Var(input[1])
  File "main_dvc.py", line 114, in Var
    return Variable(x.cuda())
AttributeError: 'str' object has no attribute 'cuda'